src/integrations/FacebookPixel/browser.js

```diff
@@ -151,16 +149,24 @@ class FacebookPixel {
     let value;
     let price;
     let query;
+    let contentName;
     if (properties) {
       products = properties.products;
       quantity = properties.quantity;
       category = properties.category;
-      prodId = properties.product_id || properties.id || properties.sku;
-      prodName = properties.product_name;
-      value = properties.value;
+      prodId = properties.product_id || properties.sku || properties.id;
+      prodName = properties.product_name || properties.name;
+      value = revValue || this.formatRevenue(properties.value);
```

Comment:
At this point `revValue` is `undefined`. Is it supposed to be?

Reply:
defined on line [121](https://github.com/rudderlabs/rudder-sdk-js/blob/d5b5de81b8605f05ecdd94bffa2e2f3b803e9720/src/integrations/FacebookPixel/browser.js#L121)